examples/common/text_data.py

```diff
@@ -218,9 +184,8 @@ def build_text_dataloader(cfg: DictConfig, device_batch_size: int):
             'remote': remote,
             'split': 'val',
             'shuffle': False,
-            'tokenizer_name': 'gpt2',
+            'tokenizer_name': 'facebook/opt-125m',
```

Comment:
I think we should probably be consistent with which tokenizer we use in examples, so maybe stick with gpt2 here (and set the separator appropriately)?